CHANGES.md

```diff
@@ -0,0 +1,17698 @@
+   - Patch releases (indicated by incrementing the PATCH number)
+     are intended for bug fixes and other improvements of existing
+     features only (like improving performance or adding documentation)
+     and retain API/ABI compatibility.
+
+   *Richard Levitte*
+
+ * Add support for RFC5297 SIV mode (siv128), including AES-SIV.
+
+   *Todd Short*
+
+ * Remove the 'dist' target and add a tarball building script.  The
+   'dist' target has fallen out of use, and it shouldn't be
+   necessary to configure just to create a source distribution.
+
+   *Richard Levitte*
+
+ * Recreate the OS390-Unix config target.  It no longer relies on a
+   special script like it did for OpenSSL pre-1.1.0.
+
+   *Richard Levitte*
+
+ * Instead of having the source directories listed in Configure, add
+   a 'build.info' keyword SUBDIRS to indicate what sub-directories to
+   look into.
+
+   *Richard Levitte*
+
+ * Add GMAC to EVP_MAC.
+
+   *Paul Dale*
+
+ * Ported the HMAC, CMAC and SipHash EVP_PKEY_METHODs to EVP_MAC.
+
+   *Richard Levitte*
+
+ * Added EVP_MAC, an EVP layer MAC API, to simplify adding MAC
+   implementations.  This includes a generic EVP_PKEY to EVP_MAC bridge,
+   to facilitate the continued use of MACs through raw private keys in
+   functionality such as EVP_DigestSign* and EVP_DigestVerify*.
+
+   *Richard Levitte*
+
+ * Deprecate ECDH_KDF_X9_62() and mark its replacement as internal. Users
+   should use the EVP interface instead (EVP_PKEY_CTX_set_ecdh_kdf_type).
+
+   *Antoine Salon*
+
+ * Added EVP_PKEY_ECDH_KDF_X9_63 and ecdh_KDF_X9_63() as replacements for
+   the EVP_PKEY_ECDH_KDF_X9_62 KDF type and ECDH_KDF_X9_62(). The old names
+   are retained for backwards compatibility.
+
+   *Antoine Salon*
+
+ * AES-XTS mode now enforces that its two keys are different to mitigate
+   the attacked described in "Efficient Instantiations of Tweakable
+   Blockciphers and Refinements to Modes OCB and PMAC" by Phillip Rogaway.
+   Details of this attack can be obtained from:
+   http://web.cs.ucdavis.edu/%7Erogaway/papers/offsets.pdf
+
+   *Paul Dale*
+
+ * Rename the object files, i.e. give them other names than in previous
+   versions.  Their names now include the name of the final product, as
+   well as its type mnemonic (bin, lib, shlib).
+
+   *Richard Levitte*
+
+ * Added new option for 'openssl list', '-objects', which will display the
+   list of built in objects, i.e. OIDs with names.
+
+   *Richard Levitte*
+
+ * Added support for Linux Kernel TLS data-path. The Linux Kernel data-path
+   improves application performance by removing data copies and providing
+   applications with zero-copy system calls such as sendfile and splice.
+
+   *Boris Pismenny*
+
+
+OpenSSL 1.1.1
+-------------
+
+### Changes between 1.1.1d and 1.1.1e [xx XXX xxxx] ###
+
+ * Added a new method to gather entropy on VMS, based on SYS$GET_ENTROPY.
+   The presence of this system service is determined at run-time.
+   [Richard Levitte]
```

Comment:
There are parts of CHANGES.md where the formatting of the author's name is not updated.

Reply:
Fixed all occurrences (hopefully) in 24ad73f8daf38f494a04ed4f5c053bcdd47b63fc.

Reply:
Second attempt: a83e6c7ebe73962d0bc4679ad3a05b6fbbc4e6d9.